src/TestSuite/Fixture/ChecksumTestFixture.php

```diff
@@ -115,7 +115,28 @@ protected function _hash(ConnectionInterface $db)
         $sth = $db->execute("CHECKSUM TABLE " . $this->table . ';');
         $result = $sth->fetch('assoc');
         $checksum = $result['Checksum'];
-        return $checksum;
+
+        // get auto increment count
+        $autoIncrement = $this->_getAutoIncrement($db);
+
+        return $checksum . $autoIncrement;
+    }
+
+/**
+ * Get the table auto increment count
+ *
+ * @param ConnectionInterface $db
+ * @return string
+ */
+    protected function _getAutoIncrement(ConnectionInterface $db)
+    {
+        $autoIncrementSth = $db->execute('SELECT `AUTO_INCREMENT` FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA=:schema AND TABLE_NAME=:table;', [
+            'schema' => $db->config()['database'],
+            'table' => $this->table,
+        ]);
+        $autoIncrementResult = $autoIncrementSth->fetch('assoc');
+
+        return (string) $autoIncrementResult['AUTO_INCREMENT'];
```

Comment:
To prevent performance degredation, we could try to execute both queries in one go.

1. Getting the table checksum
2. Fetching the ``AUTO_INCREMENT`` info

Because apart from the additional queries to execute, the biggest part of the performance degredation of this change comes most likely from the PHP side.

And since it sems it not possible to query the [table checksum](https://dev.mysql.com/doc/refman/8.4/en/checksum-table.html) as a sub query, we can only try the idea  above.

Reply:
What do you mean with "both queries in one go"? We cannot combine it like this:

```
SELECT 
  (SELECT `AUTO_INCREMENT` ...) as auto_increment,
  (CHECKSUM TABLE ...) as checksum;
```

Because the second query is not a select.

Reply:
Yeah, I tried that already.

But can we execute two queries with one `execute()` call by splitting them like `SELECT ...;CHECKSUM TABLE;`?

Reply:
Unfortunately I didn't find any way to get the result from the second query. Only the first query is returned:

```
// Debugging
$sth = $db->execute(
  "CHECKSUM TABLE " . $this->table . ';' .
  'SELECT `AUTO_INCREMENT` FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA=:schema AND TABLE_NAME=:table;',
  [
    'schema' => $db->config()['database'],
    'table' => $this->table,
  ]
);

// How to get the result of the second query?
// $sth->fetch('assoc'); returns the result of the first one.
```